lms/views/api/blackboard/files.py

```diff
@@ -38,7 +38,7 @@ def list_files(self):
         course_id = self.request.matchdict["course_id"]
 
         files = []
-        path = f"courses/uuid:{course_id}/resources?type=file&limit={PAGINATION_LIMIT}"
+        path = f"courses/uuid:{course_id}/resources?limit={PAGINATION_LIMIT}"
```

Comment:
We can probably add a `?folders` query param to our API and, here in the view, toggle the `?type=file= param depending on that param. It would be a hack, but a temporary hack, we could remove it once the frontend changes to understand folders are merged.